src/main/resources/locales/cy/translation/et1-details.json

```diff
@@ -136,5 +136,6 @@
     "Religion or belief": "Crefydd neu gred",
     "Sex": "Rhyw (gan gynnwys cyflog cyfartal)",
     "Sexual orientation": "Cyfeiriadedd rhywiol"
-  }
+  },
+  "linkToAmendTheClaim": "Welsh translation required <a href='/contact-the-tribunal/amend?lng=cy' class=\"govuk-link\">Welsh translation required</a>"
```

Comment:
Could you double check the email I sent 2nd Feb titled - FW: ET Cit UI Final batch
See if there are translations for this link

Reply:
Checked all 9 batches again, could not find the exact sentence for this text.